Finder/TranslationFinder.php

```diff
@@ -14,13 +14,19 @@ class TranslationFinder
     /**
      * @var KernelInterface
      */
-    protected $kernel;
+    private $kernel;
```

Comment:
you should probably keep `protected` here, some people use to subclass this class.